engine/link/link.asm

```diff
@@ -51,17 +51,17 @@ Gen2ToGen1LinkComms:
 	call DelayFrames
 	xor a
 	ldh [hSerialSend], a
-	ld a, (0 << rSC_ON) | (1 << rSC_CLOCK)
+	ld a, (0 << SCB_START) | (1 << SCB_SOURCE)
 	ldh [rSC], a
-	ld a, (1 << rSC_ON) | (1 << rSC_CLOCK)
+	ld a, (1 << SCB_START) | (1 << SCB_SOURCE)
 	ldh [rSC], a
 
 	call DelayFrame
 	xor a
 	ldh [hSerialSend], a
-	ld a, (0 << rSC_ON) | (1 << rSC_CLOCK)
+	ld a, (0 << SCB_START) | (1 << SCB_SOURCE)
```

Comment:
Shouldn't these `0 << *` parts be removed?

Reply:
Considering they yield the value zero, which doesn't matter in an OR, no, they aren't.